modules/nlu/src/backend/pipelines/intents/svm_classifier.ts

```diff
@@ -100,6 +106,17 @@ export default class SVMClassifier {
 
     for (const context of allContexts) {
       const intents = intentsWTokens.filter(x => x.contexts.includes(context))
+      const utterances = _.flatten(intents.map(x => x.tokens))
+      const noneUtterances = generateNoneUtterances(utterances, Math.max(5, utterances.length / 2)) // minimum 5 none utterances per context
```

Comment:
I would prefer if we generated a none intent instead of none utterance then bringing the logic of intent structure here..

We need utterances.length